ui-v2/app/styles/components/table.scss

```diff
@@ -43,6 +43,10 @@ table:not(.sessions) tbody tr {
 table:not(.sessions) td:first-child {
   padding: 0;
 }
+
+table:not(.sessions) tbody tr:hover {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
```

Comment:
How come we say not the sessions table? Is that on purpose or just a typo?

Reply:
It's on purpose, based on the pattern we don't do a lot of things on sessions.

Reply:
Ahh I getcha, these rows aren't clickable are they 👍